.github/workflows/test.yml

```diff
@@ -125,28 +125,6 @@ jobs:
         env:
           RUST_LOG: debug
 
-  kvstore-integration-latest:
-    runs-on: ubuntu-latest
-    services:
-      tendermint:
-        image: tendermint/tendermint:latest
```

Comment:
Could you not add --mode validator as an arg? this will make it work again.

Reply:
We could. But @cmwaters mentioned you'd be changing this again sometime soon - would that break it again?

Reply:
Up to now it's been a nice-to-have to test against `tendermint/tendermint:latest` in every PR (we don't fail CI if the test fails). We do, however, still want to continue testing against the latest stable releases (the `kvstore-integration-stable` test).

I was thinking we could move this `-latest` test out of this part of the CI to rather make it a daily [scheduled job](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events)?

Reply:
It will not break, the only change that is being proposed is to default to a validator instead of a fullnode

Reply:
@marbar3778 do you perhaps know how to add that parameter to a _service_ for a GitHub Action? Right now the only way I can find to override the default `start` command (replacing it with `start --mode validator`) would be to build a separate Tendermint image or come up with a complicated Docker Compose-based solution.

Just overriding the entrypoint won't work, unfortunately, since the `CMD` is defined in the Tendermint Dockerfile.